### DISCUSSION TITLE
Multiple Webdriver Instance

### DISCUSSION THREAD

**Discussion by KrAsH-CoD3** [kaliiiiiiiiii/Selenium-Profiles#51] [Q&A] [answered]
Is there a way to initialize multiple instances of selenium-profile Webdriver by async or thread?

**Comment by ttlns** [comment, marked as answer]
Async, nahh not implemented:) Don't absolutely understand it's concept yet.

But ofc possible with threading.

For multiple drivers, you can just use:
```python
def create_driver():
    profile = profiles.Windows()
    mydriver = Chrome(profile, uc_driver=False)

    driver = mydriver.start()  # or .Android
    return driver

drivers = []
for i in range(5):
    drivers.append(create_driver())
```

Note: I don't recommemd using `uc_driver=True` in that case because undetected-chromedriver seems to have issues with multiple drivers because of the way it patches chromedriver.

**Comment by ttlns** [comment]
Closing as it seems to be resolved
